test/server_helper_test.js

```diff
@@ -13,7 +13,7 @@ const gasPricePercent = 20
 
 // ServerHelper adds "noise" to shuffle requests with the same score.
 // this will prevent this randomness, to make tests deterministic.
-const noRandomness = () => 0
+const noRandomness = () => 0.5
```

Comment:
no real need to modify it, as you completely disable the test.
I will implement the randomness differently.

Reply:
There are some tests that I didn't disable. I only disabled the one test that depends on randomness